ui/src/shared/tabbedPage/TabbedPageTabs.tsx

```diff
@@ -0,0 +1,66 @@
+// Libraries
+import React, {FC} from 'react'
+import _ from 'lodash'
+
+// Components
+import {Tabs, Orientation, ComponentSize} from '@influxdata/clockface'
+import CloudExclude from 'src/shared/components/cloud/CloudExclude'
+import CloudOnly from 'src/shared/components/cloud/CloudOnly'
+import {FeatureFlag} from 'src/shared/utils/featureFlag'
+
+export interface TabbedPageTab {
+  text: string
+  id: string
+  cloudExclude?: boolean
+  cloudOnly?: boolean
+  featureFlag?: string
+  featureFlagEquals?: boolean
```

Comment:
```suggestion
  featureFlagName?: string
  featureFlagValue?: boolean
```

i think `featureFlagName` `featureFlagValue` in context with each other show that we're trying to to get the value of a feature flag by its name.